internal/jobs/pubkey_upload_aws_job.go

```diff
@@ -21,6 +21,7 @@ type PubkeyUploadAWSTaskArgs struct {
 	ARN           string `json:"arn"`
 }
 
+// Unmarshall arguments and handle error
 func HandlePubkeyUploadAWS(ctx context.Context, job dejq.Job) error {
 	ctxLogger := ctxval.Logger(ctx)
 	ctxLogger.Debug().Msg("Started pubkey upload AWS job")
```

Comment:
Saw that you removed the log msgs from Noop and Launch Instance.
SHould we remove this as well? (it is duplicated in handlePubkeyUploadAWS)

Reply:
I created a common code for all of the jobs so logging is now always guaranteed to be same.